### ISSUE TITLE
Custom training: All nan with 0 counts

### ISSUE THREAD

**Issue by omar391** [AlexeyAB/darknet#1634]
I have marked all the images with Yolo_mark tools and create a weight with yolov3.cfg.
However, when I tried to use the tiny config ( I followed this: https://github.com/AlexeyAB/darknet#how-to-train-to-detect-your-custom-objects) the training goes wrong.

![training_1](https://user-images.githubusercontent.com/13569609/45787825-2d164680-bc45-11e8-9e7c-a0191c5c3718.png)
[yolov3-tiny_obj.cfg.zip](https://github.com/AlexeyAB/darknet/files/2399069/yolov3-tiny_obj.cfg.zip)

What should I do? I have tried to check similar issues with no luck.

**Comment by omar391** [comment]
@AlexeyAB Could you please help with my issue..

**Comment by BlcaKHat** [comment]
for me also, after 900 iteration, the avg loss become 0.000 and average IOU is nan. do you have a local machine for training

**Comment by omar391** [comment]
I am using a local machine for training. I have avg loss =0, all NAN, all count =0. So it seems there must a problem somewhere.

**Comment by mohhao** [comment]
I have the same problem,have you solved it?

**Comment by omar391** [comment]
No, I am still trying. I am currently checking if any of my yolo mark annotations is wrong and then recalculate anchors,

**Comment by dexception** [comment]
Use yolo_mark tool and check your dataset.

**Comment by AlexeyAB** [comment]
What params do you use in the Makefile?

**Comment by clydebailey** [comment]
What is bad.list and bad_label.list?
